packages/backend-dev-utils/src/ipcClient.ts

```diff
@@ -61,6 +68,11 @@ export class BackstageIpcClient {
 
   constructor(sendMessage: SendMessage) {
     this.#sendMessage = sendMessage;
+    process.addListener('message', (message: unknown, _sendHandle: unknown) => {
+      if (isResponse(message)) {
+        this.#handlers.get(message.id)?.(message);
+      }
+    });
```

Comment:
Super style nit, but I'd generally prefer avoiding side effects in constructors, and instead set up the listener in the static `create()` factory. You'd add a `.handleRequest` method or something like that, which you can still access in the static factory.

Very much just a personal preference though so feel free to keep as is but with the other changes.